### DISCUSSION TITLE
希望支持自定义翻译服务接口调用比如 LibreTranslate

### DISCUSSION THREAD

**Discussion by lovedisabled** [immersive-translate/immersive-translate#2174] [Ideas]
### 新功能 | New Feature

1. 可以使用官方 LibreTranslate
2. 使用自建的 LibreTranslate 本地服务

### 为什么您想要添加这些功能 | Why you want to add these features

1. 兼容开源翻译服务，降低时延
2. 可以使用免费的服务
3. 

### 提议 | Proposal

参考 LibreTranslate 的接口文档

**Comment by seadropx** [comment]
找了一下，发现[linguist](https://github.com/translate-tools/linguist/tree/master)浏览器翻译插件支持LibreTranslate接口，初步试用一下，感觉还行。
